pyproject.toml

```diff
@@ -1,54 +1,59 @@
 [tool.poetry]
 name = "meteofrance-api"
 version = "0.1.1"
-license = "MIT"
 description = "Python client for Météo-France API."
-authors = ["oncleben31 <oncleben31@gmail.com>"]
+authors = ["oncleben31 <oncleben31@gmail.com>", "quentame <polletquentin74@me.com>", "HACF <contact@hacf.fr>"]
+license = "MIT"
+readme = "README.rst"
 homepage = "https://github.com/hacf-fr/meteofrance-api"
 repository = "https://github.com/hacf-fr/meteofrance-api"
+documentation = "https://meteofrance-api.readthedocs.io"
 keywords = ["weather", "weather-api", "meteo"]
-readme = "README.rst"
 packages = [
     { include = "meteofrance", from = "src" },
 ]
 classifiers = [
-    "Programming Language :: Python :: 3",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent",
-    "Development Status :: 3 - Alpha",
```

Comment:
I think we should keep this.

Change "Development Status" to `Development Status :: 5 - Production/Stable` perhaps

Reply:
You are right.